lib/Twig/Extensions/Grammar.php

```diff
@@ -37,3 +38,5 @@ public function getName()
         return $this->name;
     }
 }
+
+class_alias('Twig_Extensions_Grammar', 'Twig\Extensions\Grammar\AbstractGrammar', false);
```

Comment:
should be removed too, as @fabpot is deprecating the Grammar system in #205. no need to alias deprecated APIs